core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java

```diff
@@ -215,15 +215,15 @@ public DataType getDataTypeForScalarDb(
       case BIT:
         if (columnSize != 1) {
           throw new IllegalArgumentException(
-              String.format(
-                  "Data type %s(%d) is unsupported: %s", typeName, columnSize, columnDescription));
+              CoreError.JDBC_IMPORT_DATA_TYPE_WITH_SIZE_NOT_SUPPORTED.buildMessage(
+                  typeName, columnSize, columnDescription));
         }
         return DataType.BOOLEAN;
       case SMALLINT:
         if (typeName.equalsIgnoreCase("smallserial")) {
           throw new IllegalArgumentException(
-              String.format(
-                  "Data type %s(%d) is unsupported: %s", typeName, columnSize, columnDescription));
+              CoreError.JDBC_IMPORT_DATA_TYPE_WITH_SIZE_NOT_SUPPORTED.buildMessage(
```

Comment:
These cases don't seem to be related to the column size. So, JDBC_IMPORT_DATA_TYPE_NOT_SUPPORTED is better?

Reply:
The `%d` part of the message is the column size.

And, actually, `JDBC_IMPORT_DATA_TYPE_NOT_SUPPORTED` already exists.
https://github.com/scalar-labs/scalardb/pull/1493/files#diff-fd06988f4ced6a0d6d0b1049c69ab07df11c372c3ccb868a78afbe4d24e6f663R420-R421

Reply:
Yeah, my suggestion is to use JDBC_IMPORT_DATA_TYPE_NOT_SUPPORTED since the column size isn't used for the check like some others.

Reply:
Ah okay, so you mean that we should change the original error message, right? Yeah, it looks like the original one is wrong.

Reply:
Fixed in 2a51d6a639c5693dafc77b9b47bf8dcca8c3d784. I made the same change for the `serial` type. Please take a look again!